src/SwedbankPay.Sdk.Infrastructure/Extensions/HttpClientExtensions.cs

```diff
@@ -49,13 +49,15 @@ string BuildErrorMessage(string httpResponseBody)
                 httpResponseContent = await httpResponseMessage.Content.ReadAsStringAsync();
                 if (!httpResponseMessage.IsSuccessStatusCode)
                 {
+                    if (string.IsNullOrEmpty(httpResponseContent))

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)